src/styles/_guided-tour.less

```diff
@@ -97,8 +97,22 @@ div.hopscotch-bubble {
   }
 
   .hopscotch-bubble-close {
+    background: none;
+    color: transparent;
+    line-height: 0;
     right: 8px;
+    text-indent: inherit;
     top: 7px;
+    &:after {
+      content: "\e60b";
```

Comment:
Nit:  alpha.  Would be better to use @pficon-var-close as the value in case the hex values of PatternFly icons change.

Reply:
Fixed. Thanks.